tests/integrations/python/tests/test_openai.py

```
@@ -3042,3 +3042,116 @@ def test_53_web_search_streaming(self, provider, model, vk_enabled):
+        # Mean should be 5.5, std dev ~2.87
+        content_text = message.content.lower()
+        assert any(keyword in content_text for keyword in ["mean", "average", "5.5", "standard deviation", "std"]), \
+            f"Response should contain statistical results. Got: {message.content}"
+
+        print(f"✓ Code execution (data analysis) test passed!")
+        print(f"  Response: {message.content[:200]}...")
+
+    @pytest.mark.parametrize(
+        "provider,model,vk_enabled", get_cross_provider_params_with_vk_for_scenario("code_execution")
+    )
+    def test_66_code_execution_equation_solving(self, test_config, provider, model, vk_enabled):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)